bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonNetworkHandler.java

```diff
@@ -125,7 +126,7 @@ public void initialize() {
                     }, 0, DRIVER_INITIALIZED_TIME_IN_SECONDS, TimeUnit.SECONDS);
                 } else {
                     String msg = "Initialization failed, unable to start the Insteon bridge with the port '"
-                            + config.getPort() + "'.";
+                            + Utils.redactPassword(config.getPort()) + "'.";
```

Comment:
I would opt to not log this config value at all. 
As the port configuration might contain a typo, it may cause the `redactPassword` to not mask the string at all. For example all values that do not start with "/hub2/". Mistakes like "hub2/"  or "/hub/". are made easily.
Just loging a configuration error about the Port property should be enough, i guess?!

Reply:
The whole string is typically needed to help debug issues. Removing it will just make helping users more difficult.

Reply:
While it is an improvement to the current situation, i don't consider this as a full fix.
One way of dealing with it  could be seperate config parameters, but that would be a breaking change and well yeah, there does not seem to be a easy fix.